ui/tests/unit/example.spec.js

```diff
@@ -1,12 +1,8 @@
-import { shallowMount } from '@vue/test-utils';
-import HelloWorld from '@/components/HelloWorld.vue';
+// import { shallowMount } from '@vue/test-utils';
+// import HelloWorld from '@/components/HelloWorld.vue';
 
 describe('HelloWorld.vue', () => {
```

Comment:
leaving the HelloWorld.vue in is still a bit confusing.  

I think I would have preferred all of the existing test be commented out, with a note why, and then the describe() be at least a file in our code base>

Reply:
Thank you @GeraldMit. I will change describe as follows, so it's clear that this is just an example.

```
describe('TestSample.vue', () => {
  it('sample test', () => {
    expect(true);
  });
});
```

We need a general unit testing issue. #136 seems only for Selenium.